### PR TITLE
AlibabaCloud LogService Exporter - support new trace interface

### PR DESCRIPTION
**Pull request by shabicheng**
**Description:** <Describe what has changed.>

* change trace type from opencensus to opentelemetry

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

* unit test has passed with coverage 100%.
* Manually tested, sending to a LogService's instance.

**Documentation:** <Describe the documentation added.>
No need to update readme.

### CODE REVIEW

File: exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go [hunk 1]
```diff
@@ -15,393 +15,225 @@
...
-		log.Contents = append(log.Contents, tagContents...)
-	}
-	return logs
+func traceDataToLogServiceData(td pdata.Traces) ([]*sls.Log, int) {
+	var slsLogs []*sls.Log
+	resourceSpansSlice := td.ResourceSpans()
+	for i := 0; i < resourceSpansSlice.Len(); i++ {
+		resourceSpans := resourceSpansSlice.At(i)
+		if resourceSpans.IsNil() {
+			continue
```
Comment (kongluoxing) [inline exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go:53]: Should add a counter for dropped spans

Comment (shabicheng) [reply exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go:53]: resource span is never nil when used in production, so the nil should not be counted

File: exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go [hunk 2]
```diff
@@ -15,393 +15,225 @@
...
+		Key:   proto.String(traceIDField),
+		Value: proto.String(span.TraceID().HexString()),
+	})
+	contentsBuffer = append(contentsBuffer, sls.LogContent{
+		Key:   proto.String(spanIDField),
+		Value: proto.String(span.SpanID().HexString()),
+	})
+	contentsBuffer = append(contentsBuffer, sls.LogContent{
+		Key:   proto.String(parentSpanIDField),
+		Value: proto.String(span.ParentSpanID().HexString()),
```
Comment (kongluoxing) [inline exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go:111]: ParentSpanID could be empty when it's a root span, should we check it?

Comment (shabicheng) [reply exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go:112]: if ParentSpanID is not valid, the return "", it is compatible for log service

Comment (shabicheng) [reply exporter/alibabacloudlogserviceexporter/tracedata_to_logservice.go:112]: i have added a line comment about this

**Comment by shabicheng** [comment]
@bogdandrutu @tigrannajaryan Hi, could you please add my partner @kongluoxing as one of this exporter's codeowners, we will do the reviews directly in this repository.

**Comment by bogdandrutu** [comment]
@shabicheng please create a PR to add them to the codeowners file

**Comment by bogdandrutu** [comment]
@shabicheng I will expect the other owner to approve the PR at one point :)

**Review by kongluoxing — APPROVED** [review]
LGTM

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**